spi_nand_flash/include/spi_nand_flash.h

```diff
@@ -67,6 +67,14 @@ esp_err_t spi_nand_flash_copy_sector(spi_nand_flash_device_t *handle, dhara_sect
  */
 esp_err_t spi_nand_flash_write_sector(spi_nand_flash_device_t *handle, const uint8_t *buffer, dhara_sector_t sector_id);
 
+/** @brief Trim sector from the nand flash.
```

Comment:
Maybe adding a line explaining what "trim" actually means would be helpful. I didn't know this until I checked FATFS manual... Something like "Mark the sector unused, so that it can be reused by the wear levelling algorithm."

Reply:
Updated.